### PR TITLE
Sandwich builder fixes v4 807

### PR DESCRIPTION
**Pull request by cheburan**
- [x] Drop the ‘browse all foods’ sentence and instead replace it with '<StepName> categories as per sys v3

- [x] Admin: Add “Required” Option to the Admin Dashboard (locale → recipe foods → steps)

- [x] Survey: Make some steps Required (configurable via locale->recipe foods → steps

- [x] Survey: Change text for the Missing Food Prompt 

- [x] ‘I DIDN'T HAVE ANY X' for all steps won’t be possible to continue, and the system will show the message that at least one ingredient (step) is needed

### CODE REVIEW

**Comment by cheburan** [comment]
Will need to apply migrations.

File: apps/survey/src/components/prompts/portion/RecipeBuilderPrompt.vue [hunk 1]
```diff
@@ -131,10 +140,12 @@ export default defineComponent({
 
   setup() {
     const { translate } = useI18n();
+    const missingAllIngridientsDialog = ref(true);
```
Comment (lukashroch) [inline apps/survey/src/components/prompts/portion/RecipeBuilderPrompt.vue:143]: Looks like this one is not used?

File: apps/api/src/services/admin/locale.service.ts [hunk 2]
```diff
@@ -9,6 +11,7 @@ import type {
 } from '@intake24/common/types/http/admin';
 import { NotFoundError } from '@intake24/api/http/errors';
 import { addDollarSign } from '@intake24/api/util';
+import { logger } from '@intake24/common-backend/services';
```
Comment (lukashroch) [inline apps/api/src/services/admin/locale.service.ts:14]: not used

File: apps/api/src/services/admin/locale.service.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { log } from 'console';
```
Comment (lukashroch) [inline apps/api/src/services/admin/locale.service.ts:1]: not used

File: apps/admin/src/views/locales/recipe-foods/steps-dialog.vue [hunk 4]
```diff
@@ -207,6 +213,8 @@ export default defineComponent({
           return item;
         });
 
+      console.log(form.items);
```
Comment (lukashroch) [inline apps/admin/src/views/locales/recipe-foods/steps-dialog.vue:216]: redundant

**Review by lukashroch — CHANGES_REQUESTED** [review]
looks good, just couple of unused items to remove.

**Pull request merged**